packages/wonder-blocks-dropdown/components/dropdown.js

```diff
@@ -101,6 +107,12 @@ export default class Dropdown extends React.Component<Props, State> {
         };
     }
 
+    componentDidMount() {
+        if (this.props.opened != null) {
+            this.handleOpenChanged(this.props.opened);
+        }
```

Comment:
This change feels pretty orthogonal to the single-select change and could've gone in a separate PR.